### ISSUE TITLE
Add Organic Pulse views to QR WQA models

### ISSUE THREAD

**Issue by codingisforlosers** [coding-is-for-losers/seo-audit#31]

**Comment by codingisforlosers** [comment]
Added with PR https://github.com/coding-is-for-losers/qr-wqa/pull/11 on qr-wqa

**Comment by codingisforlosers** [comment]
@Mranfevb92 the organic pulse recipe includes a Sheet to tag keyword category mappings:

https://docs.google.com/spreadsheets/d/1V34xS0l-5ceaD43CbQDQT1ABcnEKazL7TR0aQGM0agA/edit#gid=1639906267

think we should build this flow into the workbook as well?

**Comment by Mranfevb92** [comment]
Its a good idea. Keyword mappings its another category that fits on the review loop we are building. 

Did you build this logic already in this [PR](https://github.com/coding-is-for-losers/qr-wqa/pull/11)? Saw there is a keyword_mapping table being used, but not sure if this going to be a manual input or how is this table going to get created in the first place.

**Comment by codingisforlosers** [comment]
Yes there’s a base table their in /admin - haven’t created the tables yet, but those can be set up in the same way as page type / actions that you set up today.

—
David Krevitt
Commissioner, CodingIsForLosers

**Comment by codingisforlosers** [comment]
Tabling this for later - let's focus first on just making sure the core WQA is adopted by people
